nixos/modules/module-list.nix

```diff
@@ -134,6 +134,7 @@
   ./services/computing/torque/server.nix
   ./services/computing/torque/mom.nix
   ./services/computing/slurm/slurm.nix
+  ./services/continuous-integration/buildbot-master/default.nix
```

Comment:
I'd probably either do ./services/continuous-integration/buildbot/master.nix (and then someday we might get a /worker.nix) or buildbot-master.nix. Most services don't use a default.nix or have supporting files.